nepalingo-web/src/components/card/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { PiSpeakerHighFill } from "react-icons/pi";
+
+const Card: React.FC<{
+  Word: string;
+  TranslatedWord: string;
+  Pronunciation: string;
+  DevenagiriSpelling: string;
+  ImageUrl: string;
```

Comment:
These types look great! A few things to account for - the ImageUrl and PronounciationUrl should be optional because not all words from the API have them available.